src/porepy/numerics/fv/mpfa.py

```diff
@@ -598,18 +620,25 @@ def _mpfa_local(g, k, bnd, eta=None, inverter="numba", apertures=None):
     bound_exclusion = fvutils.ExcludeBoundaries(subcell_topology, bnd, g.dim)
 
     # No flux conditions for Dirichlet boundary faces
-    nk_grad = bound_exclusion.exclude_dirichlet(nk_grad)
-    nk_cell = bound_exclusion.exclude_dirichlet(nk_cell)
-    # No pressure condition for Neumann boundary faces
-    pr_cont_grad_all = pr_cont_grad
-    pr_cont_grad = bound_exclusion.exclude_neumann(pr_cont_grad)
-    pr_cont_cell = bound_exclusion.exclude_neumann(pr_cont_cell)
+    nk_grad_n = bound_exclusion.exclude_rob_dir(nk_grad_all)
```

Comment:
I think somewhere we need 10 lines of documentation of the logic in the implementation

Reply:
I guess the rough idea is given for the mpfa discretization, but the discretization is messy, so it would not be bad to add some more